src/pywifes/pywifes.py

```diff
@@ -13,15 +13,17 @@
 import pylab
 import matplotlib.pyplot as plt
 import pylab
-import  astropy.units as u
+import astropy.units as u
 from pywifes.logger_config import custom_print
 import logging
+
 # Redirect print statements to logger
-logger = logging.getLogger('PyWiFeS')
-print = custom_print(logger)
-  
+logger = logging.getLogger("PyWiFeS")
+# print = custom_print(logger)
```

Comment:
Make sure to undo this before merging. When making temporary changes, best to leave a `# TODO: Undo before merging` or something along those lines, so that it can be found later.